Source/engine.h

```diff
@@ -340,6 +340,23 @@ void Cl2DrawLight(int sx, int sy, BYTE *pCelBuff, int nCel, int nWidth);
  */
 void DrawLineTo(CelOutputBuffer out, int x0, int y0, int x1, int y1, BYTE color_index);
 
+
+/**
+ * Draws a half-transparent rectangle by blacking out odd pixels on odd lines,
+ * even pixels on even lines.
+ *
+ * If blended transparency is enabled, uses `paletteTransparencyLookup` table instead
+ * of blacking pixels out.
+ *

```

Comment:
This is implementation-specific and global transparency handling is described elsewhere already.
```suggestion
 *
```